os/arch/arm/src/armv7-m/mpu.h

```diff
@@ -215,6 +220,18 @@ uint8_t mpu_log2regionfloor(uintptr_t base, size_t size);
 
 uint32_t mpu_subregion(uintptr_t base, size_t size, uint8_t l2size);
 
+/****************************************************************************
+ * Name: mpu_configure_app_regs
+ *
+ * Description:
+ *   Configure the user application SRAM mpu settings into the tcb variables
+ *
+ ****************************************************************************/
+
+#if defined(CONFIG_APP_BINARY_SEPARATION) || defined(CONFIG_MPU_STACK_OVERFLOW_PROTECTION)
+void mpu_configure_app_regs(uint32_t *regs, uint32_t region, uintptr_t base, size_t size, uint8_t readonly, uint8_t execute);
```

Comment:
The name could be confused `mpu_configure_app_regs` and `up_set_mpu_xxxx`. How about renaming `mpu_configure_app_regs`?

mpu_get_register_value
mpu_make_register_value
mpu_value2register
...

Reply:
modified the name to mpu_get_register_value, to be stored to local tcb array.